packages/web-styles/src/components/button.scss

```diff
@@ -82,95 +98,81 @@
   }
 }
 
-// Primary
-.btn-primary {
-  background-color: var(--gray-80);
-  color: var(--background-color);
+// Tertiary
+.btn-link,
+.btn-tertiary {
+  border: 0;
+  padding-right: 0.375rem;
+  padding-left: 0.375rem;
 
   @include utilities.not-disabled-focus-hover() {
-    background-color: var(--text-color);
-    color: var(--background-color); // Override <a> color
-  }
-
-  // Styles  to improve accessibility in high contrast mode
-  @include utilities.high-contrast-mode() {
-    background-color: rgba(var(--text-color-rgb), var(--button-hover-background-opacity));
-
-    &:hover {
-      border-color: highlight;
-    }
+    color: rgba(var(--text-color-rgb), 0.8);
   }
 
-  .pi,
-  &.btn-animated::after {
-    filter: invert(1);
-    forced-color-adjust: none;
+  &:disabled {
+    color: rgba(var(--text-color-rgb), 0.6);
   }
 
-  // stylelint-disable-next-line order/order
-  @include color-mx.on-dark-background() {
-    .pi {
-      filter: invert(0);
-    }
+  // Styles  to improve accessibility in high contrast mode
+  @media (forced-colors: active) {
+    color: linktext;
+    text-decoration: underline !important;
   }
 }
 
 // Icon
 .btn-icon {
-  @include icons.icon-button();
+  @include button-mx.icon-button();
 
   @each $size in sm, rg, lg {
     &.btn-#{$size} {
-      @include icons.icon-button($size);
+      @include button-mx.icon-button($size);
     }
   }
 }
 
 // Animated
-.btn-animated {
-  @include utilities.high-contrast-mode() {
-    &::after {
-      filter: invert(1);
-    }
-  }
-
+.btn-animated:not(.btn-link, .btn-tertiary) {
   &::after {
     content: '';
     position: absolute;
-    right: map.get(button.$btn-padding-x-map, rg) * (1 - button.$btn-animation-distance-factor);
-    width: 1em;
-    height: 1em;
-    transform: translateX(-2px);
+    right: button.$btn-padding-x-md - button.$btn-animation-distance-md;
+    width: 0.625em;
+    height: 0.625em;
+    transform: rotateZ(315deg) translate(-2px, -2px);
+    transform-origin: center center;
     transition: opacity 250ms, transform 250ms;
+    border-right: 2px solid currentColor;
+    border-bottom: 2px solid currentColor;
     opacity: 0;
-    background: url(icon-fn.add-stroke-color(
-      icon-fn.get-colored-svg-url(2050, color.$black),
-      color.$black,
-      2
-    ));
-    background-repeat: no-repeat;
-    background-size: contain;
+  }
+
+  @each $size in sm, rg, lg {
+    &.btn-#{$size}::after {
+      right: map.get(button.$btn-padding-x-map, $size) -
+        map.get(button.$btn-animation-distance-map, $size);
+    }
   }
 
   > span {
     transition: transform 250ms;
   }
 
-  // Only animate when user did no set animations to false
+  // Only animate when user prefers to see animations
   @media (prefers-reduced-motion: no-preference) {
```

Comment:
Is there a utilities-mixin to create this media-query?

Reply:
A mixin would provide little value since there are no additional vendor prefixes for this query. If standards are available and easy to use, I prefer standards to custom abstractions.